aws_lambda_events/Cargo.toml

```
@@ -71,6 +72,5 @@ kinesis_analytics = []
 lex = []
 rabbitmq = []
 ses = []
-sqs = []
 streams = []
 generated = ["activemq", "appsync", "autoscaling", "chime_bot", "clientvpn", "cloudwatch_logs", "code_commit", "codebuild", "codedeploy", "codepipeline_cloudwatch", "codepipeline_job", "cognito", "config", "connect", "ecr_scan", "firehose", "iot", "iot_1_click", "iot_button", "kafka", "kinesis", "kinesis_analytics", "lex", "rabbitmq", "s3", "s3_batch_job", "ses", "sqs", "streams"]
```

you need to remove it from the `generated` array too
```suggestion
generated = ["activemq", "appsync", "autoscaling", "chime_bot", "clientvpn", "cloudwatch_logs", "code_commit", "codebuild", "codedeploy", "codepipeline_cloudwatch", "codepipeline_job", "cognito", "config", "connect", "ecr_scan", "firehose", "iot", "iot_1_click", "iot_button", "kafka", "kinesis", "kinesis_analytics", "lex", "rabbitmq", "s3", "s3_batch_job", "ses", "streams"]
```